$NetBSD$

Fix the assumption that python3 does exist as a binary,
in a less ideal way.
Upstream ticket: https://github.com/rtbo/rust-xcb/issues/62

--- ../vendor/xcb-0.8.2/build.rs~	2019-07-10 09:03:56.574458875 +0000
+++ ../vendor/xcb-0.8.2/build.rs	2019-07-10 09:05:41.085255414 +0000
@@ -64,7 +64,7 @@
 
         if ref_mtime > src_file_mtime || ref_mtime > ffi_file_mtime {
 
-            let status = Command::new("python3")
+            let status = Command::new("python3.7")
                     .arg(&r_client)
                     .arg("-o").arg(&src_dir)
                     .arg(&xml_file)
